agent/src/main/java/com/cloud/agent/properties/AgentProperties.java

```diff
@@ -39,14 +39,6 @@
      */
     public static final Property<Integer> VM_MIGRATE_DOMAIN_RETRIEVE_TIMEOUT = new Property<Integer>("vm.migrate.domain.retrieve.timeout", 10);
 
-    /**
-     * Reboot host and alert management on heartbeat timeout. <br>
-     * Data type: boolean.<br>
-     * Default value: true.
-     */
-    public static final Property<Boolean> REBOOT_HOST_AND_ALERT_MANAGEMENT_ON_HEARTBEAT_TIMEOUT
-        = new Property<Boolean>("reboot.host.and.alert.management.on.heartbeat.timeout", true);
```

Comment:
If we gonna remove this property, we need to explicit the option in the new feature that behave the same way and advise operators on how to configure it.

Also, we should remove from `agent.properties` too and wherever it is used.

Reply:
The default action is to reboot the system using the -c flag. so i will just remove it from agent.properties file